src/routers/secure/scrape.py

```
@@ -441,4 +450,49 @@ async def complete_manual_session(_: Request, session_id: str) -> SessionRespons
+    return {"message": f"Completed session {session_id}"}
+
+@router.post(
+    "/scrape/upload_torrent",
+    summary="Upload a torrent file to start a manual scraping session",
+    operation_id="upload_torrent_file"
+)
+async def upload_torrent_file(
+    request: Request,
+    background_tasks: BackgroundTasks,
+    item_id: str,
+    torrent_file: UploadFile = File(...),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)